page-objects/pages/DashboardPage.js

```diff
@@ -14,7 +24,58 @@ async function isTitleOfDashboard() {
 	await waitHelpers.waitUntilPageTitleContains(dashboardBtnOnNavigationLeftPanel);
 }
 
+async function isPrivateTasksAndDraftsListVisible(listName) {
+	const dashboardTitlesList = await elementHelpers.getTextFromAllElements(taskElement.dashboardTitleList)
+	const msg = 'Title with name [' + listName + '] is not visible on the list \n';
+	const resultMsg = ', List of titles: ' + dashboardTitlesList;
+	return expect(dashboardTitlesList, msg + resultMsg).to.include(listName);
+}
+
+async function isCourseNameOnPrivateTaskVisible(taskName, courseName) {
+	const taskIndex = await getIndexOfTaskFromList(taskName);
+	const taskCourseElementList = await elementHelpers.getListOfAllElements(taskElement.courseName)
+	const courseNameText = await elementHelpers.getElementText(taskCourseElementList[taskIndex]);
+	const msg = 'Course with name [' + courseName + '] is not visible on the private task \n';
+	const resultMsg = 'List of course titles: ' + taskCourseElementList;
+	return expect(courseNameText, msg + resultMsg).to.include(courseName);
+}
+
+async function getIndexOfTaskFromList(taskName) {
+	const taskTextElementList = await elementHelpers.getListOfAllElements(taskElement.taskName)
+	const taskTextValueList = await elementHelpers.getTextListFromListOfElements(taskTextElementList);
+	const index = taskTextValueList.indexOf(taskName);
+	return index;
+}
+
+async function isPrivateTaskNameVisible(taskName) {
+	const taskNameList = await elementHelpers.getTextFromAllElements(taskElement.taskName)
+	const msg = 'Task with name [' + taskName + '] is not visible on the list \n';
+	const resultMsg = 'List of Task titles: ' + taskNameList;
+	return expect(taskNameList, msg + resultMsg).to.include(taskName);
+}
+
+async function isElementOnTaskVisible(elementName, taskName, selector, expectedValue) {
+	const defaultString = `Element with name: ${elementName}`;
+	let elementOfTask = true;
+	const taskIndex = await getIndexOfTaskFromList(taskName);
+	const elementsOfTaskList = await elementHelpers.getListOfAllElements(selector);
+	elementsOfTaskList.length === 0 ? elementOfTask = false : await waitHelpers.waitUntilElementIsPresent(elementsOfTaskList[taskIndex]);
```

Comment:
Do I see correctly that you don't save to the variable the value returned if the condition is false? Why? Will you not use it later?